### PR TITLE
Adds contributing.md file

### PR DESCRIPTION
**Pull request by kathyoleary**
:green_book: Ready to go...need a few more thumbs...

### CODE REVIEW

**Comment by ahaynes-ador** [comment]
I have some copyediting: https://github.com/adorableio/LMHT/pull/5

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+## Edit Documentation
+
+There are two ways you can submit a change to documentation.  The [Issues page] can be used as well as [email].  Either way, make a copy of the documentation, edit it-clearly marking the changes you are requesting-and submit it.  Both email and the Issues page allow you to copy the document into your submission.
+
+[Issues page]: https://github.com/LMHT/LMHT.github.io/issues
+[email]: mailto:lmht@adorable.io
+
+## Become a GitHub Contributor
+
+If you wish to become a contributor to LMHT, please send an email to lmht@adorable.io that will include basic contact information, what areas you wish to contribute to (areas of expertise) such as english to spanish translation, coding, level of comfort using github.com, etc so that we can get your access set up.  As a contributor, the expectation is that you can use github fluently enough to fork branches, send pull requests and basically maneuver through github to send changes to update or correct the lmht pages as needed.
```
Comment (ahaynes-ador) [inline contributing.md]: Why do we need an email with all this information from people? If they're making contributions on GitHub, isn't it safe to assume these people know how to use it? (This is not a rhetorical question.)

Also, we don't have to grant anyone access to let them make PRs. It's a public project.

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+* [Request a Feature](#request-a-feature)
+* [Edit Documentation](#edit-documentation)
+* [Become a GitHub Contributor](#become-a-github-contributor)
+* [Approval Process](#approval-process)
+
+
+
+## Ways to Contribute
+
+There are several ways to contribute to LMHT depending on your level of github experience and what you want to accomplish:
```
Comment (ericanaughton) [inline contributing.md]: should github be capitalized?

File: contributing.md (hunk 1 above)
Comment (ericanaughton) [inline contributing.md]: Good point @LipGlossary. @kathyoleary @JenEEsquivel This document (Contributing.md) will only be visible on Github, in the repo. So people who see it already know how to use Github. The email system is meant for users who do not have experience on Github, and they have questions or a desire to contribute. So the Welcome Section on our live site is the appropriate place to mention details about the email expectations. It won't hurt to mention our email address on this document or the Readme, though.

Comment (ericanaughton) [inline contributing.md]: and just some nit picky things that i think should be capitalized:
LMHT
Github
English
Spanish

**Comment by ericanaughton** [comment]
@kathyoleary how's it going on the changes for this?

**Comment by kathyoleary** [comment]
Updated with Annalise's changes and then did a few more of my own and put back out there for two more thumbs up before merging this.

**Comment by ericanaughton** [comment]
@bigtiger said this is good to go. :+1:

**Pull request merged**